src/components/docs-product-logo.js

```diff
@@ -1,13 +1,22 @@
 import React from "react"
 import getCloudflareDocsConfig from "../utils/get-cloudflare-docs-config"
+import getUniqueReadableID from "../utils/get-unique-readable-id"
 import AccessibleSVG from "./accessible-svg"
 
+const titleID = getUniqueReadableID("title")
+
 export default () => {
-  const { product, productLogoPathD }  = getCloudflareDocsConfig()
+  const { product, productLogoPathD, productIconKey, logoSVGContent } = getCloudflareDocsConfig()
 
-  return (
+  return logoSVGContent ? (
+    <span
+      id={titleID}
```

Comment:
What's the point of this ID? Won't `titleID` be the same for every instance of this `<span>`

Reply:
well it generates a random ID for the span that holds the svg?

Reply:
it was being used in the AccesivleSVG component so I also added it here

Reply:
just removed it !